src/main/java/dev/jbang/cli/Edit.java

```diff
@@ -101,7 +102,11 @@ public Integer doCall() throws IOException {
 					cmd = new String[] { "cmd", "/c", editorCommand };
 				}
 				verboseMsg("Running `" + String.join(" ", cmd) + "`");
-				new ProcessBuilder(cmd).start();
+				ProcessBuilder pb = new ProcessBuilder(cmd);
+				Path javaHome = JdkManager.getCurrentJdk(ctx.getJavaVersionOr(ss));
+				pb.environment().put("JAVA_HOME", javaHome.toString());
+				pb.environment().put("PATH", javaHome.resolve("bin") + File.pathSeparator + System.getenv("PATH"));
```

Comment:
what if java is found in /usr/bin won't this be wrong ?

Reply:
I'll re-check but AFAIK `getCurrentJdk()` always returns a valid Path to the JDK that was found, not just JBang's managed JDKs.

Reply:
But you were right, that case had to be handled separately.